.github/workflows/deploy.yml

```diff
@@ -72,9 +69,13 @@ jobs:
 
           # Use Docker `latest` tag convention
           [ "$VERSION" == "develop" ] && VERSION=latest
+          
+          #Date Time
+          $DATETIME=$(date +%s)
 
           echo IMAGE_ID=$IMAGE_ID
           echo VERSION=$VERSION
+          echo DATETIME=$DATETIME
 
-          docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-          docker push $IMAGE_ID:$VERSION
+          docker build . --file Dockerfile --tag $IMAGE_ID:$DATETIME --tag $IMAGE_ID:$DATETIME
+          docker push $IMAGE_ID
```

Comment:
For the master branch, can we please push a `latest` tag as well please

Reply:
My bad I pushed the same tag twice.